catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -68,18 +72,27 @@
 
     private SlotTypeHelper slotTypeHelper = new SlotTypeHelper();
 
+    private ScheduledExecutorService executorService;
+
     private InternationalStringTypeHelper internationalStringTypeHelper =
             new InternationalStringTypeHelper();
 
-    public void init() throws PrivilegedActionException {
-        Security.runAsAdminWithException(() -> {
-            if (!federationAdminService.getLocalRegistryIdentityMetacard()
-                    .isPresent()) {
-                createIdentityNode();
-            }
-            return null;
-        });
+    public void init() {
+        try {
+            Security.runAsAdminWithException(() -> {
+                if (!federationAdminService.getLocalRegistryIdentityMetacard().isPresent()) {
+                    createIdentityNode();
+                }
+                return null;
+            });
+        } catch (PrivilegedActionException e) {
+            LOGGER.warn("Error checking for local registry identity node. Will try again later");
+            executorService.schedule(this::init, RETRY_INTERVAL, TimeUnit.SECONDS);
```

Comment:
The case where this will always fail is while the catalog isn't up. In that case the system isn't in a state to do much. Should the retry happen until it does come up? If it never comes up, things are probably coming down soon.